documentation/cli/change_log.rst

```diff
@@ -7,6 +7,7 @@ FlexMeasures CLI Changelog
 since v0.26.0 | June 03, 2025
 =================================
 * Switch to ``flexmeasures jobs save-last`` CLI command for saving the last n canceled/deferred/failed/finished/scheduled/started jobs (from the scheduling queue, by default).
+* Add ``flexmeasures show assets`` CLI command for listing public assets and option ``--account <account-id>`` to list assets owned by a specific account.
```

Comment:
0.26 has been released already. We should file this under 0.27 (new section).

Also please add a changelog entry in our main changelog.